src/spinner.tsx

```diff
@@ -5,6 +5,7 @@ import FadeIn from './fade-in';
 import {useAriaId, useTheme} from './hooks';
 import * as styles from './spinner.css';
 import {vars} from './skins/skin-contract.css';
+import {useIsInverseVariant} from './theme-variant-context';
 
 type Props = {
     color?: string;
```

Comment:
imo, this prop should be removed😛. should we show it in story controls?

Reply:
If we don't want to support custom colors in spinner, we should deprecate the prop and remove it in the next major

Reply:
Discussed offline. This is tricky, there are scenarios where you actually need to set a color. For example, in ButtonLinkDanger the spinner is red.

I'm afraid that this may break other scenarios, and I don't see as something bad that the color is customizable.

Given all this, we decided to align design specs in order to formally accept the customizable color: https://github.com/Telefonica/mistica-design/issues/1791

Reply:
lgtm